.github/workflows/build_latest.yml

```diff
@@ -0,0 +1,77 @@
+name: Latest Qt5
```

Comment:
Maybe change the name to something like "Use generated code", as this is certainly not the latest Qt5. There are two othyer workflows (ubuntu 22.04 and WinGW 7.3) with Qt 5.15.

Reply:
Yes, there is some misunderstanding between "latest qt" and 5.12 :)
I will rename it to the more suitable form